servlet-map/src/main/resources/service-workers/xhr-prioritizer.js

```diff
@@ -0,0 +1,73 @@
+const DEBUG = false;
+const MAX_CONCURRENT_REQUESTS = 6;
+const LIMIT_CONCURRENT_REQUESTS = 2;
+const PRIO_TRAFFIC_LIMIT = 3;
```

Comment:
Remember this doesn’t go through babel so it’s best to use es5

Reply:
Downgraded to ES5. Left fetch intact because it's more widely supported than service workers.